pkg/minishift/clusterup/proxyutil.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright (C) 2017 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package clusterup
+
+import (
+	"regexp"
+	"strings"
+)
+
+//Parses the proxy URI and returns a slice ["proxy_hostname:proxy_port", "user", "password"]
+func ParseProxyUri(proxyUri string) []string {
```

Comment:
I think it should be 

    func ParseProxyUri(proxyUri string) (string, string, string, error)